### ISSUE TITLE
Unable to register from desktop app, should be disabled

### ISSUE THREAD

**Issue by Extarys** [vector-im/riot-web#7848]
### Description

When a user register an account on the Desktop version of Riot, it says the Robot verification is not yet compatible and the user need to click on a link that open a web page.

The user then need to re-enter it's registration information.

There is no way int he app to go back, you need to close the app and re-open it.

The experience is awful and should be corrected.

### Steps to reproduce

Create an acocunt on the desktop app

Describe how what happens differs from what you expected.

I would expect to
 1) be able to make the captcha in the app (I understand though why this isn't possible), or;
 2) Tell the user registrations are currently not supported in the desktoop version,
or at least
 3) put a button to go back to the login screen
### Version information

- **Platform**: desktop

For the desktop app:
matrix-react-sdk version: <local>
riot-web version: 0.17.8
olm version: 3.0.0
- **OS**: Linux Mint
- **Version**: 0.17.8

**Comment by Extarys** [comment]
When I first tried it a couple of days ago, I gave up.

- I tried creating an account on riot-android
- Received some email
- Try clicking on the links to follow the steps
- Didn't work
- Installed the desktop app
- Fill in registration info
- Being told I need to open a browser
- Refill my registration information

It was quite painful - I was very sad and told myself I could never try and switch my friends to Riot. But I decided to make a little video tutorial for them. :D

UX is an important part to attract people now.

Thanks for your reply.
